docs/index.html

```diff
@@ -209,6 +209,42 @@ <h1>Electronics and electrical design checklist</h1>
 				</li>
 			</ul>
 		</li>
+		<li>EMC
+			<ul>
+				<li> Requirements
+					<ul>
+						<li> What noise freqencies are emitted? (e.g. clock and SMPS frequencies) </li>
```

Comment:
I've never seen the abbreviation SMPS before!

I think it would be good to include abbreviations like this:

```suggestion
						<li> What noise freqencies are emitted? (e.g. clock and <abbr title="Switched-mode power supply">SMPS</abbr> frequencies) </li>
```

There are a lot of abbreviations already in the document and they might be new to many people, so I'll try to adopt this style in the future. Thanks for making me realize it.

Reply:
Agreed, I haven't seen the `<abbr>` tag before!